docs/02-app/01-building-your-application/06-configuring/03-environment-variables.mdx

```diff
@@ -168,3 +168,4 @@ For example, if `NODE_ENV` is `development` and you define a variable in both `.
 ## Good to know
 
 - If you are using a [`/src` directory](/docs/app/building-your-application/configuring/src-directory), `.env.*` files should remain in the root of your project.
+- The environment variable `NODE_ENV` is automatically assigned to `development` when running `next dev`. All other commands automatically assign `NODE_ENV` to `production`.
```

Comment:
We don't override `test` if set x-ref: https://github.com/vercel/next.js/blob/e10431612dbb4ee5df72622ab4b01bab1013b52a/packages/next/src/bin/next.ts#L84

Reply:
Looks like we don't override at all. We just have defaults.

Reply:
@ijjk Updated in [ce2933f](https://github.com/vercel/next.js/pull/52003/commits/ce2933f3ce94345da9583c610d638d1af459b7fe)